router/config/config.go

```diff
@@ -226,6 +230,7 @@ type Config struct {
 	LocalhostFallbackInsideDocker bool                        `yaml:"localhost_fallback_inside_docker" default:"true" envconfig:"LOCALHOST_FALLBACK_INSIDE_DOCKER"`
 	CDN                           CDNConfiguration            `yaml:"cdn"`
 	DevelopmentMode               bool                        `yaml:"dev_mode" default:"false" envconfig:"DEV_MODE"`
+	NATS                          NATSConfiguration           `yaml:"nats"`
```

Comment:
I thought we want to keep it generic e.g. adding kafka support later. I would also describe it from a feature perspective e.g.

```yaml
eventing:
	provider: "NATS"
	url: "..."
```

Reply:
I've refactored it to use a more generic config format, which should allow us to support multiple data source types moving forward. PTAL